src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -111,6 +111,25 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
+@include breakpoint($small-screen) {
+    // JUST FOR TESTING ******** REMOVE BEFORE PUSHING ----
+    .bdl-CollapsibleSidebar-wrapper.hidden {
```

Comment:
Can you tell us what this is for and what happens when you remove it?

Reply:
I put this in there to show the wrapper. Is there a ff or cookie on or something? Anyway, it was a quick fix to see the sidebar